diff -up plasma-phone-components-master/bin/kwinwrapper.in.omv~ plasma-phone-components-master/bin/kwinwrapper.in
--- plasma-phone-components-master/bin/kwinwrapper.in.omv~	2020-12-21 05:24:44.826127011 +0100
+++ plasma-phone-components-master/bin/kwinwrapper.in	2020-12-21 05:24:49.691192965 +0100
@@ -24,4 +24,4 @@ if [ -d @KDE_INSTALL_FULL_LIBDIR@/libhyb
 fi
 
 touch /tmp/simplelogin_starting
-startplasma-wayland --xwayland --libinput --lockscreen --inputmethod maliit-keyboard --exit-with-session=@KDE_INSTALL_FULL_LIBEXECDIR@/startplasma-waylandsession
+startplasma-wayland --xwayland --libinput --inputmethod maliit-keyboard --exit-with-session=@KDE_INSTALL_FULL_LIBEXECDIR@/startplasma-waylandsession
