diff -up plasma-phone-components-5.22.0/bin/kwinwrapper.in.2~ plasma-phone-components-5.22.0/bin/kwinwrapper.in
--- plasma-phone-components-5.22.0/bin/kwinwrapper.in.2~	2021-06-04 18:53:21.049024329 +0200
+++ plasma-phone-components-5.22.0/bin/kwinwrapper.in	2021-06-04 18:58:58.394186757 +0200
@@ -16,4 +16,4 @@ export QT_VIRTUALKEYBOARD_STYLE=Plasma
 
 export PLASMA_DEFAULT_SHELL=org.kde.plasma.phoneshell
 
-startplasma-wayland --xwayland --libinput --lockscreen --inputmethod maliit-keyboard --exit-with-session=@KDE_INSTALL_FULL_LIBEXECDIR@/startplasma-waylandsession
+startplasma-wayland --xwayland --libinput --inputmethod maliit-keyboard --exit-with-session=@KDE_INSTALL_FULL_LIBEXECDIR@/startplasma-waylandsession
