diff -up plasma-phone-components-master/bin/plasma-mobile.desktop.cmake.omv~ plasma-phone-components-master/bin/plasma-mobile.desktop.cmake
--- plasma-phone-components-master/bin/plasma-mobile.desktop.cmake.omv~	2020-08-27 20:50:55.778641881 +0200
+++ plasma-phone-components-master/bin/plasma-mobile.desktop.cmake	2020-08-27 20:51:01.323709809 +0200
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Exec=dbus-run-session ${CMAKE_INSTALL_FULL_BINDIR}/kwinwrapper
+Exec=${CMAKE_INSTALL_FULL_BINDIR}/kwinwrapper
 TryExec=${CMAKE_INSTALL_FULL_BINDIR}/kwinwrapper
 DesktopNames=KDE
 Name=Plasma Mobile
