diff -up plasma-phone-components-6204b90a5106e1a8acb1853efa0c8555d7c3097c/bin/kwinwrapper.in.4~ plasma-phone-components-6204b90a5106e1a8acb1853efa0c8555d7c3097c/bin/kwinwrapper.in
--- plasma-phone-components-6204b90a5106e1a8acb1853efa0c8555d7c3097c/bin/kwinwrapper.in.4~	2021-01-08 04:19:50.899107048 +0100
+++ plasma-phone-components-6204b90a5106e1a8acb1853efa0c8555d7c3097c/bin/kwinwrapper.in	2021-01-08 04:20:16.975599114 +0100
@@ -16,6 +16,6 @@ export QT_VIRTUALKEYBOARD_STYLE=Plasma
 
 export GRID_UNIT_PX=25
 export FORCE_RIL_NUM_MODEMS=1
-export PLASMA_DEFAULT_SHELL=org.kde.plasma.phoneshell
+export PLASMA_DEFAULT_SHELL=org.kde.plasma.phone
 
 startplasma-wayland --xwayland --libinput --inputmethod maliit-keyboard --exit-with-session=@KDE_INSTALL_FULL_LIBEXECDIR@/startplasma-waylandsession
diff -up plasma-phone-components-6204b90a5106e1a8acb1853efa0c8555d7c3097c/CMakeLists.txt.4~ plasma-phone-components-6204b90a5106e1a8acb1853efa0c8555d7c3097c/CMakeLists.txt
--- plasma-phone-components-6204b90a5106e1a8acb1853efa0c8555d7c3097c/CMakeLists.txt.4~	2021-01-04 08:39:32.000000000 +0100
+++ plasma-phone-components-6204b90a5106e1a8acb1853efa0c8555d7c3097c/CMakeLists.txt	2021-01-08 04:19:50.899107048 +0100
@@ -41,7 +41,7 @@ find_package(KF5 ${KF5_MIN_VERSION} REQU
 include(CheckIncludeFiles)
 
 plasma_install_package(look-and-feel org.kde.plasma.phone look-and-feel)
-plasma_install_package(shell org.kde.plasma.phoneshell shells)
+plasma_install_package(shell org.kde.plasma.phone shells)
 
 install(DIRECTORY kwinmultitasking/ DESTINATION ${DATA_INSTALL_DIR}/kwin/scripts/org.kde.phone.multitasking)
 install(FILES kwinmultitasking/metadata.desktop DESTINATION ${SERVICES_INSTALL_DIR} RENAME kwin-script-org.kde.phone.multitasking.desktop)
diff -up plasma-phone-components-6204b90a5106e1a8acb1853efa0c8555d7c3097c/shell/metadata.desktop.4~ plasma-phone-components-6204b90a5106e1a8acb1853efa0c8555d7c3097c/shell/metadata.desktop
--- plasma-phone-components-6204b90a5106e1a8acb1853efa0c8555d7c3097c/shell/metadata.desktop.4~	2021-01-04 08:39:32.000000000 +0100
+++ plasma-phone-components-6204b90a5106e1a8acb1853efa0c8555d7c3097c/shell/metadata.desktop	2021-01-08 04:19:50.899107048 +0100
@@ -79,7 +79,7 @@ X-KDE-PluginInfo-Author=Aaron Seigo
 X-KDE-PluginInfo-Category=
 X-KDE-PluginInfo-Email=aseigo@kde.org
 X-KDE-PluginInfo-License=GPLv2+
-X-KDE-PluginInfo-Name=org.kde.plasma.phoneshell
+X-KDE-PluginInfo-Name=org.kde.plasma.phone
 X-Plasma-FallbackPackage=org.kde.plasma.nano
 X-KDE-PluginInfo-Version=0.1
 X-KDE-PluginInfo-Website=https://plasma-mobile.org
